docs/book/src/tasks/experimental-features/cluster-class-how-to.md

```diff
@@ -0,0 +1,3 @@
+# How to write a ClusterClass
```

Comment:
Nit:
should we move everything related to ClusterClass under ./tasks/experimental-features/cluster-class/, so we can drop cluster class prefix from the files (cluster-classes.md could be renamed in overview.md or something similar)

Also, wondering if we have a clear definition of what is an "how-to" and what are "operations". A thing that we can do is to just drop the distinction and have focused pages on each topic